### DISCUSSION TITLE
How do I change just the title of the downgrade error dialog box?

### DISCUSSION THREAD

**Discussion by ricardoleonhart** [wixtoolset/issues#6532] [Questions] [answered]
Currently, I am able to change the title of all error dialog boxes in Wix by using this piece of XML in the .wxl file

```
<WixLocalization Culture="en-us"  Language="1033" xmlns="http://schemas.microsoft.com/wix/2006/localization">
  <!-- Change title to Testing 123 for all error dialog boxes-->
  <String Id="ErrorDlg_Title">Testing 123</String>
</WixLocalization>
```
However, I am unable to find out how to change the title just for a specific dialog box namely the [Downgrade Error dialog box.](https://i.stack.imgur.com/aJEXE.png). I have checked out the `<MajorUpgrade />` element and it only exposes the DowngradeErrorMessage property for any kind of message customization on this dialog box.

**Question**

Is there a way to change just the title of the Downgrade Error dialog box?

**Comment by robmen** [comment, marked as answer]
Off the top of my head, I think that is implemented as a Windows Installer error which takes its title from the `Product/@Name` attribute. Not sure it can be customized. But that is old memory and I could be mistaken.
